allauth/account/adapter.py

```diff
@@ -563,9 +563,20 @@ def is_safe_url(self, url):
 
         return url_has_allowed_host_and_scheme(url, allowed_hosts=allowed_hosts)
 
+    def request_password_reset(self, users):
```

Comment:
Thoughts:
- Naming -- it does not request the password reset
- Behavior -- if you filter users here the emails are silently skipped which is not very obvious.

Wouldn't it make be more useful to add this method to the adapter?:

```
def send_password_reset_mail(self, user, email)
```

Then, you are free to implement that as you see fit (send real reset, skip the mail, or send another mail).